README.md.tmpl

```diff
@@ -0,0 +1,225 @@
+# GoAT: Go ASCII Tool
+<!--
+  NOTE to maintainers
+  ---
+    SVG examples/ regeneration.
+       go test -run . -v -write
+
+    Github home page README.md, specific to $USER:
+       sed "s,{{\.Root}},https://cdn.rawgit.com/${USER}/goat/main," README.md.tmpl >README.md
```

Comment:
Interesting -- was this helpful for your debugging/testing?

Reply:
> ... transparency, but it can let some of the arrow tails peek through ...

I see your point, and I've looked at the code enough to appreciate how hard a fix that aligns the stroke end to the edge of the circle could be.  How about I add a command-line boolean switch, defaulting to the old behavior?

Reply:
Regarding the **sed** script to generate README.md, this is useful for making output-altering changes visible on the code fork front page.

I have some such changes on the way, but I'll attend to coding and pushing the above command-line switch to draw opaque background, first.

Reply:
> I see your point, and I've looked at the code enough to appreciate how hard a fix that aligns the stroke end to the edge of the circle could be. How about I add a command-line boolean switch, defaulting to the old behavior?

I think the easiest fix would be to use some masking in the SVG output to subtract polygons from arrows/lines.